services/core-api/app/api/email_preview/resources/email_preview_resource.py

```diff
@@ -0,0 +1,230 @@
+                        'project_title': 'Mine Life Extension Project',
+                        'submitted': 'September 25, 2024'
+                    },
+                    'project_section': 'Environmental Assessment',
+                    'core_link': 'https://core.gov.bc.ca/project/456/section/789',
+                    'minespace_link': 'https://minespace.gov.bc.ca/project/456'
+                }
+                return {**base_data, **template_data}
+            else:  # project summary
+                template_data = {
+                    'message': 'New Project Description Submitted',
+                    'mine': {
+                        'mine_name': 'Mountain View Mine',
+                        'mine_no': 'MV-456789'
+                    },
+                    'project_summary': {
+                        'project_summary_description': 'Expansion of existing copper mining operations with new processing facility'
+                    },
+                    'core_project_summary_link': 'https://core.gov.bc.ca/projects/456',
+                    'minespace_project_summary_link': 'https://minespace.gov.bc.ca/projects/456',
+                    'ema_auth_link': 'https://ema.gov.bc.ca/auth'
+                }
+                return {**base_data, **template_data}
+                
+        elif 'mine_party_appt' in template_name:
+            template_data = {
+                'start_date': '2024-01-15',
+                'party': {
+                    'first_name': 'John',
+                    'last_name': 'Engineer'
+                },
+                'tsf_name': 'Tailings Storage Facility A',
+                'mine': {
+                    'mine_name': 'Test Mine',
+                    'mine_no': 'TM001'
+                },
+                'submitted_at': '2024-01-15 10:30:00',
+                'core_appt_link': 'https://core.gov.bc.ca/appointments/123',
+                'minespace_appt_link': 'https://minespace.gov.bc.ca/appointments/123',
+                'minespace_login_link': 'https://minespace.gov.bc.ca/login'
+            }
+            return {**base_data, **template_data}
+            
+        else:
+            # Default test data for unknown templates
+            template_data = {
+                'message': 'Test Email Template',
+                'mine_name': 'Test Mine',
+                'mine_no': 'TEST001',
+                'link': 'https://example.com'
+            }
+            return {**base_data, **template_data}
+
+
+class EmailPreviewListResource(Resource, UserMixin):
+    """
+    List all available email templates
+    """
+    
+    @requires_any_of([VIEW_ALL, MINESPACE_PROPONENT])
+    def get(self):
+        """
+        List all available email templates for preview
+        GET /api/email-preview/
+        """
+        try:
+            import os
+            from pathlib import Path
+            
+            templates_dir = Path(current_app.root_path) / 'templates' / 'email'
```

Comment:
Would suggest using `os.path.join` instead of `/` here (in case anyone decides to try this on windows 🪟)